<launch>
	<param name="scan_topic" type="str" value="/lidar2d/scan" />
    <node pkg="prizm_control" type="move_to_object" name="move_to_object" output="screen" launch-prefix="gnome-terminal -e"/>
</launch>
